zyte_common_items/components.py

```diff
@@ -38,6 +38,7 @@ class _ListMetadata(Item):
     #: Date and time when the product data was downloaded, in UTC timezone and
     #: the following format: ``YYYY-MM-DDThh:mm:ssZ``.
     dateDownloaded: Optional[str] = None
+    validation: Optional[dict] = None
```

Comment:
Could you add documentation for the field? (with `#:`-prefixed comments like above)